CHANGELOG.md

```diff
@@ -1,5 +1,31 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2023-09-03
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
+
+* No new clusters can be created with [Azure AD Integration (legacy)](https://learn.microsoft.com/azure/aks/azure-ad-integration-cli). Existing AKS clusters with Azure Active Directory integration will keep working. All Azure AD Integration (legacy) AKS clusters will be migrated to [AKS-managed Azure AD](https://learn.microsoft.com/azure/aks/managed-azure-ad) automatically starting from 1st Dec. 2023. We recommend updating your cluster with AKS-managed Azure AD before 1 Dec 2023. This way you can manage the API server downtime during non-business hours.
+* OSsku=cblmariner (Mariner 1.0) reached end of life on July 31, and is no longer receiving security updates, in accordance with Mariner's 2 year lifecycle for major versions. Please migrate to OSsku=azurelinux (Mariner 2.0). Read more on [Mariner 2.0][https://microsoft.github.io/CBL-Mariner/announcing-mariner-2.0/]
+* To avoid disruptions stemming from unmanaged Canonical nightly security updates, AKS will disable unmanaged Canonical nightly updates by 2 September 2023
+on clusters that haven’t specified an update option explicitly, mapping to the option `None` in the [node OS upgrade](https://learn.microsoft.com/azure/aks/auto-upgrade-node-image) channel feature. AKS strongly recommends proactively moving to [auto-upgrade node-image](https://learn.microsoft.com/azure/aks/auto-upgrade-cluster) or [node OS upgrade channel - SecurityPatch](https://learn.microsoft.com/azure/aks/auto-upgrade-node-image); you can set [maintenance windows](https://learn.microsoft.com/azure/aks/planned-maintenance) for these channels.
+
+### Release notes
+  
+* Behavioral changes
+  * [Outbound IP][https://learn.microsoft.com/en-us/azure/aks/outbound-rules-control-egress] can now be a combination of ip/ipprefix and managed ones.
+  * Added v1 version for Group Managed Service Accounts (gmsacredentialspec) CRDs. [Reference][https://github.com/kubernetes-sigs/windows-gmsa/commit/2f2f42f1a31fce00caf74b85d6a7b52f9eec2797]
```

Comment:
* GMSA can now be disabled on an existing cluster. https://learn.microsoft.com/azure/aks/use-group-managed-service-accounts